src/widgets/Footer/Footer.tsx

```diff
@@ -1,3 +1,21 @@
+import svgSprite from '../../shared/Icons/icons.svg';
+
 export const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="flex justify-center items-center bg-primary-10 ">
+      Footer
+      {
+        <>
```

Comment:
Yes, it will removed.
This was only an example you had to consult. Can we use icon sprites in such a way or should we create a special Component for them?